selfservice/hook/web_hook_test.go

```diff
@@ -0,0 +1,212 @@
+package hook
```

Comment:
Could you add integration tests with `httptest.NewServer` for the web-hook receiver with different config options? :)

Reply:
Done